packaging/centos7/cloud.spec

```diff
@@ -437,6 +437,12 @@ if [ -f "%{_sysconfdir}/cloud.rpmsave/agent/agent.properties" ]; then
     mv %{_sysconfdir}/cloud.rpmsave/agent/agent.properties %{_sysconfdir}/cloud.rpmsave/agent/agent.properties.rpmsave
 fi
 
+# Enable TLS enabled VM migration for libvirtd
+if ! iptables-save | grep -- "-A INPUT -p tcp -m tcp --dport 16514 -j ACCEPT" > /dev/null; then
+    iptables -t filter -A INPUT -p tcp -m tcp --dport 16514 -j ACCEPT
+    iptables-save > /etc/iptables/rules.v4
+fi
```

Comment:
And here again

Reply:
We'll document in release notes and admin/install docs. Fixed.